packages/terra-clinical-item-collection/src/_ItemCollectionUtils.jsx

```diff
@@ -0,0 +1,79 @@
+const KEYCODES = {
+  ENTER: 13,
+  SPACE: 32,
+};
+
+/**
+ * This function ensures the correct elements are provided to create a consistent item view layout. To handle a
+ * potential missing required accessory element, a boolean indicator is provided to single accessory space is needed.
+ *
+ * @ param {childElement} obj - The react child props that will compose an item view list item.
```

Comment:
It sounds like these are childProps and parentProps if I'm reading this correctly?

Reply:
Could you expand on your statement? I'm not sure what you mean by parent props. But I from what I think you are asking, 

requriedElements is an object that passes the `hasStartAccessory`, `numberOfDisplays`, `hasComment` and `hadEndAccessory` prop values of the Item Collection component

and 

childElements are the element pieces (start/end accessory, comment and displays) of each ItemCollection Item.